prometheus_client/exposition.py

```diff
@@ -17,13 +17,14 @@
 )
 from wsgiref.simple_server import make_server, WSGIRequestHandler, WSGIServer
 
+from packaging.version import Version
+
 from .openmetrics import exposition as openmetrics
 from .registry import CollectorRegistry, REGISTRY
 from .utils import floatToGoString
-from .validation import _is_valid_legacy_metric_name
 
 __all__ = (
-    'CONTENT_TYPE_LATEST',
+    'CONTENT_TYPE_PLAIN',
```

Comment:
This needs to stay as `CONTENT_TYPE_LATEST` as well. We could add in other versions if we choose to go the version route in the options below.